application/org.openjdk.jmc.jolokia/plugin.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--   
+   Copyright (c) 2018, 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I won't comment regarding copyright in any more files, but the comment before of course applies to all of them.